src/Http/Routing.Abstractions/src/LinkGenerator.cs

```diff
@@ -118,7 +118,7 @@ public abstract class LinkGenerator
         /// </summary>
         /// <typeparam name="TAddress">The address type.</typeparam>
         /// <param name="address">The address value. Used to resolve endpoints.</param>
-        /// <param name="values">The route values. Used to expand parameters in the route template. Optional.</param>
+        /// <param name="values">The route values. Used to expand parameters in the route template.</param>
```

Comment:
The documentation claims this is optional, but later code expects this to be a non-null value.